python/cudf/cudf/core/column/string.py

```diff
@@ -4859,6 +4862,50 @@ def edit_distance(self, targets) -> ParentType:
             cpp_edit_distance(self._column, targets_column)
         )
 
+    def edit_distance_matrix(self) -> ParentType:
+        """Computes the edit distance between strings in the series.
+
+        The series to compute the matrix should have more than 2 strings and
+        should not contain nulls.
+
+        Edit distance is measured based on the Levenshtein edit distance
+        algorithm.
+        https://www.cuelogic.com/blog/the-levenshtein-algorithm
+
+        Parameters
+        ----------
+        None, does not require input parameter.
```

Comment:
```suggestion
        Edit distance is measured based on the `Levenshtein edit distance
        algorithm
        <https://www.cuelogic.com/blog/the-levenshtein-algorithm>`_.

```

Can remove `Parameters` as there are none, and also pack the url into the text.